app/models/common_page.rb

```diff
@@ -0,0 +1,54 @@
+# Page model for pages with content shared by all modules
```

Comment:
Love this and the deprecation of the two pages - thanks for cleaning this up :) - I might have gone a step further and put these all in a folder/namespace.

Reply:
Are you thinking along the lines of `Pages::Common`, `Pages::Content`, `Pages::Video`?

Reply:
Perhaps if we switch to Contentful we could do the new content in the Pages namespace.